Funcky.Money.SourceGenerator/Iso4217RecordGenerator.cs

```diff
@@ -86,7 +86,7 @@ private static string GenerateAllCurrenciesProperty(IReadOnlyCollection<Iso4217R
         {
             var property = new StringBuilder();
 
-            property.AppendLine($"{Indent}{Indent}internal static IReadOnlyCollection<Currency> AllCurrencies {{ get; }} = ImmutableArray.Create(");
+            property.AppendLine($"{Indent}{Indent}internal static ImmutableArray<Currency> AllCurrencies {{ get; }} = ImmutableArray.Create(");
```

Comment:
I want to access Length (which would in IReadOnlyCollection) and the index operator (which would be in IReadOnlyList) so we need to expose the type here.

Reply:
There's `IReadOnlyList`…

Reply:
Adressed in 320b99b5